packages/components/src/Illustration/Scene/BrandMomentCaptureIntro/_docs/BrandMomentCaptureIntro.stories.tsx

```diff
@@ -0,0 +1,43 @@
+import { Meta, StoryObj } from "@storybook/react"
+import { BrandMomentCaptureIntro } from "../index"
+
+const meta = {
+  title: "Components/Illustrations/BrandMomentCaptureIntro",
```

Comment:
Should it be `Illustrations/Scene/Brand...`?

Reply:
hmm yeah, its a strange one. Technically it is its own thing, but logically it would make better sense to group it in there since it's only ever used as a stand-in for scene